locale/en.po

```diff
@@ -5873,13 +5868,13 @@ msgstr ""
 
 msgid "TUTORIAL_MICROBE_EDITOR_ENDING_TEXT"
 msgstr ""
-"Those are the basics of editing your species. As a finishing touch, you can rename your species by clicking on the name on the bottom left and editing the text.\n"
+"Adaptation is hard. There are many sources of [thrive:compound type=\"atp\"][/thrive:compound], but not all of them are in every patch. You can use all the tabs and panels here to make the most informed choice you can, or just try something and see how it works!\n"
```

Comment:
I think if we are skipping the patch map, a tutorial that is seen before the patch map should not refer to patches at all as the player will have no clue what patches are.